script/check_static

```diff
@@ -3,9 +3,7 @@
 checks = [
   'script/check_gemfile',
   'script/check_contributors_md',
-  'script/check_conflicts',
-  'rubocop --display-cop-names --rails',
-  "haml-lint app/views"
+  'script/check_conflicts'
```

Comment:
This defeats the idea of "you can run all the static checks with one command". Please either move the call to `overcommit -r` in here, or kill this script and add the ad-hoc checks to overcommit (and change the "how to run all the tests" instructions in `CONTRIBUTING.md` accordingly).